gen/app-gen

```diff
@@ -25,12 +25,13 @@ Simple tool to generate Mender Artifact suitable for App Update Module
 
 Usage: $0 [options] [-- [options-for-mender-artifact] ]
 
-    Options: [ -n|--artifact-name -t|--device-type -o|--output_path --i|--image -p|--platform -r|--orchestrator -m|--manifests-dir -h|--help -a|--application-name ]
+    Options: [ -n|--artifact-name -t|--device-type -o|--output_path -i|--image -d|--deep-delta -p|--platform -r|--orchestrator -m|--manifests-dir -h|--help -a|--application-name ]
```

Comment:
Nit: `-i` should be a separate commit

Reply:
I will try, but chances are, I will just create a one commit of it all.